evalml/objectives/standard_metrics.py

```diff
@@ -365,6 +365,7 @@ class MAPE(TimeSeriesRegressionObjective):
     """
     name = "Mean Absolute Percentage Error"
     greater_is_better = False
+    score_needs_proba = False
```

Comment:
To allow `MAPE` to fall under the same `objective_class.rst` as the other metrics